bin/deploy-release

```diff
@@ -25,6 +25,9 @@ echo "::endgroup::"
 cluster_name=$(terraform -chdir="infra/${app_name}/service" output -raw service_cluster_name)
 service_name=$(terraform -chdir="infra/${app_name}/service" output -raw service_name)
 echo "Wait for service ${service_name} to become stable"
-aws ecs wait services-stable --cluster "${cluster_name}" --services "${service_name}"
+if ! aws ecs wait services-stable --cluster "${cluster_name}" --services "${service_name}"; then
+  echo "First attempt to wait for service stability failed, retrying..."
+  aws ecs wait services-stable --cluster "${cluster_name}" --services "${service_name}"
```

Comment:
nit: I'd maybe define a little function for this, rather than repeating the long command with parameters each time, but the instances are right next to each other, so not a big deal IMO.

Reply:
good idea